### PR TITLE
Elm 0.19 upgrade

### PR DESCRIPTION
**Pull request by MattCheely**
This updates ember-elm to work for Elm 0.19. It's based on the other PR I submitted with formatting changes, so if that merges, the diffs here should shrink quite a bit. Or if it's declined, I'll rebase on my end and update the PR. 

I also updated the error message formatting to try to make it look a bit nicer when running ember locally:

ember-elm-error

### CODE REVIEW

**Comment by nucleartide** [comment]
D: @MattCheely, there are some merge conflicts – would you be able to resolve them? 🙏

**Comment by MattCheely** [comment]
My mistake! I'll get 'em fixed.

File: tests/dummy/app/elm-modules/Chat.elm [hunk 1]
```diff
@@ -1,76 +1,97 @@
 port module Chat exposing (main)
 
+import Browser
 import Html exposing (..)
 import Html.Attributes exposing (..)
 import Html.Events exposing (..)
-import WebSocket as WS
```
Comment (MattCheely) [inline tests/dummy/app/elm-modules/Chat.elm]: Websocket support is not in 0.19 yet, so my only option was to remove the import. Makes this program less interesting, but it still compiles.

Comment (nucleartide) [reply tests/dummy/app/elm-modules/Chat.elm]: Looks good to me!

**Comment by MattCheely** [comment]
Thanks for the merge! If you have time to publish a new release, I'd be grateful. :pray:

**Pull request merged**